test/update-version-test.js

```diff
@@ -0,0 +1,106 @@
+"use strict";
+
+const grunt = require( "grunt" );
+const runTask = require( "grunt-run-task" );
+let tempFilePath = [ "tmp/test.php", "tmp/README.MD" ];
+let expectedFilePath = [ "test/expected/test.php", "test/expected/README.MD" ];
+let noOfFiles = Math.min( tempFilePath.length, expectedFilePath.length );
```

Comment:
If this refers to 'number of files', please name this variable as such (i.e. `numberOfFiles`).